cmd/disco/VERSION

```diff
@@ -1 +1 @@
-1.0.2
+1.0.3
```

Comment:
Not a big thing, but I think it might be worth capturing this in the release notes.